orchestra/migrations/0083_update_staffbot_request_status_choices.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.9 on 2020-03-30 19:21
+
+from django.db import migrations, models
+
+def migrate_request_status(apps, schema_editor):
```

Comment:
how long does the migration take to run?

Reply:
About 40s.